dor-services.gemspec

```diff
@@ -24,7 +24,8 @@ Gem::Specification.new do |s|
   s.add_dependency 'json', '~> 1.8.1'
   s.add_dependency 'net-sftp', '~> 2.1.2'
   s.add_dependency 'net-ssh',  '~> 2.6.5' # 2.6.5 is pegged by our 12/2014 version of net-stfp
-  s.add_dependency 'nokogiri', '~> 1.6.0'
+  s.add_dependency 'rsolr', '~> 1.0'
```

Comment:
Instead of pinning rsolr, can you backport https://github.com/sul-dlss/dor-services/commit/c5ca8fb71cfe267b1c85127b81e22320a3dc9bb3?